BedrockServer.cpp

```diff
@@ -2047,9 +2060,10 @@ void BedrockServer::onNodeLogin(SQLitePeer* peer)
             for (const auto& fields : table) {
                 cmd->crashIdentifyingValues.insert(fields.first);
             }
-            auto _syncNodeCopy = atomic_load(&_syncNode);
-            if (_syncNodeCopy) {
-                _syncNodeCopy->broadcast(_generateCrashMessage(cmd), peer);
+            auto _clusterMessengerCopy = _clusterMessenger;
+            if (_clusterMessengerCopy) {
+                BedrockCommand cppTypesAreHard(_generateCrashMessage(cmd), nullptr);
+                _clusterMessengerCopy->runOnPeer(cppTypesAreHard, peer->name);
```

Comment:
Honest to god could not think of a better variable name today, open to suggestions. We already use `command` and `cmd` and this is all type juggling

Reply:
Thank you, I was blanking so hard when I wrote this, this is great 😆